edx_solutions_api_integration/courses/views.py

```diff
@@ -459,6 +460,7 @@ def get(self, request, course_id, content_id):
         data_blocks = get_blocks(
             request,
             usage_key,
+            user=request.user,
```

Comment:
^ Same here. I don't think we need this changed for now, and I don't think this makes any difference as this is likely just `AnonymousUser`.